README.md

```diff
@@ -8,41 +8,32 @@ of [sass][2].
 [1]: https://github.com/dart-lang/build
 [2]: https://github.com/sass/dart-sass
 
-## Attention: Transformer has been removed in v2.0.0.
-
-> The transformer provided by this package has been removed. The `pub build` and
-> `pub serve` commands have been replaced by `build_runner` in version
-> 2.0.0-dev.51.0 of the Dart SDK.
->
-> * [Announcement](https://groups.google.com/a/dartlang.org/forum/#!topic/announce/R4kV3us0Sm8)
-> * [Migration Guide](https://webdev-dartlang-org-dev.firebaseapp.com/dart-2#tools)
-
 ## Usage
 
 1\. Create a `pubspec.yaml` file containing the following code:
 
 ```yaml
 dependencies:
-    # update to the latest version
-    bootstrap_sass: any
+  # (optional) depend on the latest version of packages providing sass sources
+  bootstrap_sass: any
 dev_dependencies:
-    # update to the latest versions
-    sass_builder: ^2.1.2
-    build_runner: ^1.0.0
+  # update to the latest versions
+  sass_builder: ^2.1.2
+  build_runner: ^2.1.7
 ```
 
 2\. Create `web/main.scss` containing the following code:
 
 ```scss
-@import "sub";
-@import "package:bootstrap_sass/scss/variables";
+@use "sub";
+@use "package:bootstrap_sass/scss/variables" as bootstrap;
```

Comment:
I'm not sure if `@use "package:..."` will automatically work like `@import` with these dart package style imports. We should add tests similar to those we have for `@import "package:..."` this before we recommend `@use`.

Reply:
Good point, I've added a test importing both relative and package urls through `@use`.